packages/wallet-service/tests/txProcessor.test.ts

```diff
@@ -1,441 +1,19 @@
-  // receive another block, for a different address
-  block.tx_id = 'txId3';
-  block.timestamp += 10;
-  block.height += 1;
-  block.outputs = [createOutput(0, blockReward, 'address2')];
-  await txProcessor.onNewTxEvent(evt);
-  // we now have 3 blocks and 2 addresses
-  await expect(checkUtxoTable(mysql, 3, 'txId3', 0, '00', 'address2', blockReward, 0, null, block.height + blockRewardLock, true)).resolves.toBe(true);
-  await expect(checkAddressTable(mysql, 2, 'address2', null, null, 1)).resolves.toBe(true);
-  await expect(checkAddressTxHistoryTable(mysql, 3, 'address2', 'txId3', '00', blockReward, block.timestamp)).resolves.toBe(true);
-  // new block reward is locked
-  await expect(checkAddressBalanceTable(mysql, 2, 'address2', '00', 0, blockReward, null, 1)).resolves.toBe(true);
-  // address1's balance is all unlocked now
-  await expect(checkAddressBalanceTable(mysql, 2, 'address1', '00', 2 * blockReward, 0, null, 2)).resolves.toBe(true);
-  expect(await getLatestHeight(mysql)).toBe(block.height);
-
-  // spend first block to 2 other addresses
-  const tx = evt.Records[0].body;
-  tx.version = 1;
-  tx.tx_id = 'txId4';
-  tx.timestamp += 10;
-  tx.inputs = [createInput(blockReward, 'address1', 'txId1', 0)];
-  tx.outputs = [
-    createOutput(0, 5, 'address3'),
-    createOutput(1, blockReward - 5, 'address4'),
-  ];
-  await txProcessor.onNewTxEvent(evt);
-  expect(await getLatestHeight(mysql)).toBe(block.height);
-  for (const [index, output] of tx.outputs.entries()) {
-    const { token, decoded, value } = output;
-    // we now have 4 utxos (had 3, 2 added and 1 removed)
-    await expect(checkUtxoTable(mysql, 4, tx.tx_id, index, token, decoded.address, value, 0, decoded.timelock, null, false)).resolves.toBe(true);
-    // the 2 addresses on the outputs have been added to the address table, with null walletId and index
-    await expect(checkAddressTable(mysql, 4, decoded.address, null, null, 1)).resolves.toBe(true);
-    // there are 4 different addresses with some balance
-    await expect(checkAddressBalanceTable(mysql, 4, decoded.address, token, value, 0, null, 1)).resolves.toBe(true);
-    await expect(checkAddressTxHistoryTable(mysql, 6, decoded.address, tx.tx_id, token, value, tx.timestamp)).resolves.toBe(true);
-  }
-  for (const input of tx.inputs) {
-    const { decoded, token, value } = input;
-    // the input will have a negative amount in the address_tx_history table
-    await expect(checkAddressTxHistoryTable(mysql, 6, decoded.address, tx.tx_id, token, (-1) * value, tx.timestamp)).resolves.toBe(true);
-  }
-  // address1 balance has decreased
-  await expect(checkAddressBalanceTable(mysql, 4, 'address1', '00', blockReward, 0, null, 3)).resolves.toBe(true);
-  // address2 balance is still locked
-  await expect(checkAddressBalanceTable(mysql, 4, 'address2', '00', 0, blockReward, null, 1)).resolves.toBe(true);
-});
-
-test.skip('txProcessor should be able to re-process txs that were voided in the past', async () => {
```

Comment:
In the new daemon we don't need to re-process transactions, we now handle the `metadataChanged` event and void the utxos as needed. We have tests for this in `packages/daemon/__tests__/services/services.test.ts`